AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -554,8 +560,19 @@ - (void)applyPipelineEnabledState:(BOOL)isEnabled {
 - (void)initializeChannelGroup {
 
   // Construct channel group.
-  self.oneCollectorChannelDelegate =
-      self.oneCollectorChannelDelegate ?: [[MSOneCollectorChannelDelegate alloc] initWithInstallId:self.installId];
+  if (self.oneCollectorChannelDelegate) {
```

Comment:
This implementation doesn't seem correct, Blocking the PR for now until I review this again. Sorry for delay.

Reply:
hey, no worries!

Reply:
OK

Reply:
This `if` statement doesn't seem required. You don't need to call `setLogUrl` here. Let's remove.